docs/scenarios_authoring/Intro.md

```diff
@@ -17,23 +17,23 @@ Nussknacker nodes process events; once the node finishes processing of the event
 Typically events processed by Nussknacker come from Kafka topics;  Nussknacker source components are used to read events from Kafka topic and inject them into Nussknacker scenarios. 
 There are cases though when a node can produce a new event, a Tumbling-window or and Session-window component being good examples. 
 
-## Notion of time
+## (Streaming-Flink only) Notion of time
 
 Notion of passing time is very important in dealing with real time events processing. 
 Please see following excellent references to learn about basic concepts:
 * [Notion of time in Flink](https://ci.apache.org/projects/flink/flink-docs-stable/docs/concepts/time/)
 * [Streaming 101: The world beyond batch by O'Reilly](https://www.oreilly.com/radar/the-world-beyond-batch-streaming-101/)
 
-Nussknacker uses Flink, so in general, Flink documentation applies. Certain Nussknacker components make assumptions and have predefined settings, so that the end users don't have to configure all by themselves.
+For Nussknacker's Flink engine Flink documentation applies. Certain Nussknacker components make assumptions and have predefined settings, so that the end users don't have to configure all by themselves.
```

Comment:
How about "For Nussknacker's Flink engine Flink documentation applies." -> "For Streaming-Flink engine Flink documentation applies."

Reply:
fixed